recipes/himalaya/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.30.0"
+
+
+class HimalayaConan(ConanFile):
+    name = "himalaya"
+    license = "GPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Himalaya-Library/Himalaya"
+    description = "C++ library to calculate three-loop corrections to the Higgs boson mass in the MSSM"
+    topics = ("conan", "high-energy", "physics", "hep", "Higgs", "mass", "MSSM")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
```

Comment:
```suggestion
    options = {"fPIC": [True, False], "shared": [True, False]}
    default_options = {"fPIC": True, "shared": False}
```

The cmake file lists Himalaya_shared and Himalaya_static

Reply:
I think the option `shared` does not make sense, because Himalaya always builds a shared and a static library. Or am I misunderstanding your proposal?

Reply:
You can't package both at same package. You have to package them separately.

Reply:
Ok, thanks! :) Would adding the `shared` option automatically package the right library (static/shared) that has been built?

Reply:
Nope, `shared` sets `BUILD_SHARED_LIBS` cmake definition, but it useless for Himalaya, as it builds both.

Reply:
Ok. Should your proposed changes applied anyway?

Reply:
Yes, I just want to understand gfortran error first.